### PR TITLE
feat: include jvm build

### LINKED ISSUE #6: Generate JAR files for the library
**Description**
Although we have the `ios` and `android` module set up, we need to expect the library to generate files as well for use on desktop JAVA since the AAR files cannot be understood by then. 

**Solution**
We may need another module set up similar to `ios` or `android` that can be outputted as `jar`. 

**Alternative**
N/A

### PR DESCRIPTION
**Pull request by BumbleFlash**
Integrates JVM build to generate JAR files. 

"Closes #6"

### CODE REVIEW

**Comment by BumbleFlash** [comment]
@barbeau Any suggestions on where do I write tests? 
Should I create a compose desktop similar to BikeShare and write unit tests with in?

**Review by barbeau — COMMENTED** [review]
Thanks @BumbleFlash! In the readme could you also please add the commands and output path for the JAR (and which JAR to use if you're bundling it manually) in the "Desktop Java/Kotlin" section?

**Review by barbeau — APPROVED** [review]
Thanks @BumbleFlash!

File: library/build.gradle.kts [hunk 1]
```diff
@@ -7,10 +7,23 @@ plugins {
     id("com.android.library")
     id("kotlinx-serialization")
     id("kotlin-android-extensions")
+    id("maven-publish")
+}
+
+group = "edu.usf.cutr.otp"
+version = "1.0.0"
+
+repositories {
```
Comment (barbeau) [inline library/build.gradle.kts:16]: @BumbleFlash Was there a reason that a dependency on `mavenLocal()` was needed as part of this PR? Looking at this now (as part of #12), I don't think it's needed. Normally you'd have this in a project that was trying to use the library. You would run `./gradlew publishToMavenLocal` in this project, and then add the `mavenLocal()` in the application that depends on the library.

Comment (BumbleFlash) [reply library/build.gradle.kts:16]: Yeah, I don't think we need it as you said. I can remove it in the next PR.

**Pull request merged**